helm/cert-manager/charts/cert-manager-giantswarm-clusterissuer/values.yaml

```diff
@@ -46,4 +46,4 @@ acme:
       secretAccessKey: ""
   http01:
     enabled: true
-    ingressClassName: nginx
+    ingressClass: ""
```

Comment:
For my intertest, what will be the outcome of removing the default?
Will it just result the solver ingresses to have no class or will they have the class of the original ingress? Could there be a negative outcome for cases in which other issuers/classes are defined?

Reply:
I came to the conclusion that this change to "" or default doesn't makes sense for now, as there may be no ingress controllers deployed by the time cert-manager is deployed